### PR TITLE
Add application user to the Dockerfile

### PR DESCRIPTION
**Pull request by ewelinajozwik**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
 
 # non-overridable environment variables
-ENV APP_CONFIG_DIR=gb-backend.config.yml
+ENV APP_USR gb_backend_usr
+ENV APP_USR_HOME "/home/${APP_USR}"
+ENV APP_CONFIG_DIR ${APP_USR_HOME}/gb-backend.config.yml
 
-COPY entrypoint.sh /entrypoint.sh
+COPY docker/entrypoint.sh /entrypoint.sh
+COPY gb-backend/VERSION /VERSION
```
Comment (ewelinagr) [inline docker/Dockerfile]: @gijskant Should it use the version file from gb-backend or should there be a separate file for docker? Or better to keep using the variable (previous way) for now?

File: docker/Dockerfile [hunk 2]
```diff
@@ -1,19 +1,30 @@
...
-ENV GB_BACKEND_WAR_URL=https://repo.thehyve.nl/content/repositories/releases/nl/thehyve/gb-backend/${GB_BACKEND_VERSION}/gb-backend-${GB_BACKEND_VERSION}.war
+ENV APP_PORT 8083
+ENV GB_BACKEND_VERSION 0.4.0-RC1
+ENV GB_BACKEND_WAR_URL https://repo.thehyve.nl/content/repositories/releases/nl/thehyve/gb-backend/${GB_BACKEND_VERSION}/gb-backend-${GB_BACKEND_VERSION}.war
 
 # non-overridable environment variables
-ENV APP_CONFIG_DIR=gb-backend.config.yml
+ENV APP_USR gb_backend_usr
+ENV APP_USR_HOME "/home/${APP_USR}"
+ENV APP_CONFIG_DIR ${APP_USR_HOME}/gb-backend.config.yml
```
Comment (gijskant) [inline docker/Dockerfile]: Maybe this is the right time to rename APP_CONFIG_DIR to APP_CONFIG_FILE?

File: docker/Dockerfile [hunk 3]
```diff
@@ -1,19 +1,30 @@
...
-# download the war
-RUN wget "${GB_BACKEND_WAR_URL}" -O "gb-backend-${GB_BACKEND_VERSION}.war" && \
+EXPOSE ${APP_PORT}
+
+USER root
+
+RUN adduser -h "${APP_USR_HOME}" -D ${APP_USR} && \
     sed -i 's/\r//' /entrypoint.sh && \
-    chmod +x /entrypoint.sh
+    chown -R "${APP_USR}" "${APP_USR_HOME}" && \
```
Comment (gijskant) [inline docker/Dockerfile]: Is this needed if the homedir is created by the adduser command two lines earlier?

**Review by gijskant — APPROVED** [review]
Looks good; added some minor comments

**Pull request merged**